services/app-web/src/pages/StateSubmission/ReviewSubmit/RateDetailsSummarySection.tsx

```diff
@@ -282,9 +280,24 @@ export const RateDetailsSummarySection = ({
                           rateFormData.deprecatedRateProgramIDs.length > 0
                       const hasNoRatePrograms =
                           rateFormData.rateProgramIDs.length === 0
+                      const isLinkedRate = rate.isLinked
+
+                      /*
+                    Rate programs switched in summer 2024. We still show deprecated program field values when
+                    - there's no new field values present and CMS user is viewing
+                    - theres no new field values present and contract is locked and state user viewing
+                    - theres no new field values present and the contract unlocked and the rate being displayed as a linked rate
+
+                    otherwise use new fields values going forward
+                    */
+                      const showLegacyRatePrograms =
+                          hasDeprecatedRatePrograms &&
+                          hasNoRatePrograms &&
+                          (isSubmittedOrCMSUser || isLinkedRate)
```

Comment:
Not sure if we care enough, but once a submission is unlocked going to the submission summary page, as a state user, will show the inline error for the new program field. You can get to the submission summary page when it is unlocked by using the path `/submissions/:id`. We don't expose a way to get to this path in the app itself as a State user, but I think some emails have this link and maybe savvy users.

Reply:
🐛 Uh, that seems like a bug, you aren't supposed to be able to view the submission summary page for an unlocked rate - you are supposed to get redirected to review/submit to edit